src/transformers/models/whisper/generation_whisper.py

```diff
@@ -260,6 +264,7 @@ def generate(
         language: Optional[str] = None,
         is_multilingual: Optional[bool] = None,
         prompt_ids: Optional[torch.Tensor] = None,
+        prompt_condition_type: Optional[str] = None,  # first-segment, all-segments
```

Comment:
(nit) can go in the docstring
```suggestion
        prompt_condition_type: Optional[str] = None,
```

Reply:
I'd prefer keeping the comment here actually as this way one can quickly see what options are allowed